biorbd_optim/limits/path_conditions.py

```diff
@@ -242,6 +261,32 @@ def concatenate(self, other):
         self.min = PathCondition(np.concatenate((self.min, other.min)), interpolation=self.min.type)
         self.max = PathCondition(np.concatenate((self.max, other.max)), interpolation=self.max.type)
 
+        self.type = self.min.type
+        self.t = self.min.t
+        self.extra_params = self.min.extra_params
+        self.nb_shooting = self.min.nb_shooting
+
+    def __getitem__(self, slice_list):
+        if isinstance(slice_list, slice):
+            t = self.min.t
+            param = self.extra_params
+            interpolation = self.type
+            if interpolation == InterpolationType.CUSTOM:
+                min_bound = self.min.custom_function
+                max_bound = self.max.custom_function
+            else:
+                min_bound = np.array(self.min[slice_list.start : slice_list.stop : slice_list.step])
+                max_bound = np.array(self.max[slice_list.start : slice_list.stop : slice_list.step])
```

Comment:
This seems redundant with what is done in Plot? I haven't double check. If so, don't do it by hand in the plot, but use the slicing you just coded!

Reply:
This logic should not be in Bounds, but in PathCondition